vault/dendron.tutorial.md

```diff
@@ -2,28 +2,31 @@
 id: 678c77d9-ef2c-4537-97b5-64556d6337f1
 title: Getting Started
 desc: ''
-updated: 1631317031955
+updated: 1639425805599
 created: 1608051264282
 nav_order: 1.1
 ---
 
-
 ## Pre-requisites
 
-This walkthrough will help you get started with Dendron. Before you start, make sure you do the following:
+This walkthrough will help you get started with the basics of Dendron. Before you start, make sure you do the following:
+
 - installed [VSCode](https://code.visualstudio.com/download) or a VSCode compatible editor like [VSCodium](https://vscodium.com/)
-- installed [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the code marketplace
-- initialize your first Dendron workspace
+- installed [Dendron](https://link.dendron.so/vscode) from the code marketplace
+- initialize your first Dendron **[[workspace|dendron://dendron.dendron-site/dendron.topic.workspace]]**
+
+![[dendron://dendron.dendron-site/dendron.topic.workspace#summary,1:#*]]
```

Comment:
can we update the summary? some issues with the current summary:
- it introduces new concepts that users don't know about yet (eg. `Dendron: Initialize Workspace`)
- it introduces multiple concepts (eg. workspace, workspace root, vaults, dendron.yml)

```md
In Dendron, your **workspace** is the root of where all your files are located. It's set when you first run `Dendron: Initialize Workspace`. The folder that contains your workspace is also known as your **workspace root**.
A workspace has a `dendron.yml` file that specifies its [[vaults|dendron.topic.vaults]].
```

update
```md
## Summary

A Dendron workspace is a collection of one or more [[vaults|dendron.topic.vaults]] (folders with notes). 

## Details

Every workspace has a [[dendron.yml|dendron://dendron.dendron-site/dendron.ref.config]] which specifies all vaults and configuration for a particular workspace

Most workspaces only have a single vault. See [[Multi Vault|dendron://dendron.dendron-site/dendron.topic.multi-vault]] for scenarios of adding additional vaults.

## Concepts

### Workspace Root

The folder that contains `dendron.yml` is known ...
```